Bottom: 1965425b91bd69ac4973e511a85f99a5137da5ee
Top:    65e5876d017a4d2251ca06cbae9980c932e4a3db
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-10 15:59:53 +0100

Refresh of fixup-depth-for-layers-and-ele

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index ec3957e..4fcd961 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1898,8 +1898,8 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
     int max_phys_group;
     int i;
 
-    solder_group = GetLayerGroupNumberByNumber (max_layer + SOLDER_LAYER);
-    component_group = GetLayerGroupNumberByNumber (max_layer + COMPONENT_LAYER);
+    solder_group = GetLayerGroupNumberByNumber (solder_silk_layer);
+    component_group = GetLayerGroupNumberByNumber (component_silk_layer);
 
     min_phys_group = MIN (solder_group, component_group);
     max_phys_group = MAX (solder_group, component_group);
